stan/math/prim/cplx/complex.hpp

```diff
@@ -0,0 +1,57 @@
+#ifndef STAN_MATH_PRIM_CPLX_COMPLEX_HPP
+#define STAN_MATH_PRIM_CPLX_COMPLEX_HPP
+
+#include <stan/math/prim/cplx/zeroing.hpp>
+#include <complex>
+
+namespace stan {
+namespace math {
+
+/**
+ *  Complex class that forwards the interface of std::complex, brining
```

Comment:
what is meant here? I still don't understand what "forwards" means in the way it's used here.

Reply:
The class inherits from std::complex, so it behaves as std::complex does. This is necessary because this class has more flexible constructors that prevent some of the problems mentioned further down, but still must perform the duties of the real complex class also.